src/EventConsumerService.cpp

```diff
@@ -51,11 +51,29 @@ void EventConsumerService::processAccessoryEvent(VlcbMessage *msg, unsigned int
   }
 }
 
+void EventConsumerService::process(UserInterface::RequestedAction requestedAction)
+{
+  if (coeService && (coeService->available()))
+  {
+    // DEBUG_SERIAL << ">Getting COE Message " << endl;
+    VlcbMessage *msg = coeService->get();
+    bool done = handleEventMessage(msg);
+    // DEBUG_SERIAL << ">COE Message handled " << endl;
+  }
+}
+
 Processed EventConsumerService::handleMessage(unsigned int opc, VlcbMessage *msg) 
 {
+  return handleEventMessage(msg);
+}
+
+Processed EventConsumerService::handleEventMessage(const VlcbMessage *msg)
```

Comment:
The "const" causes problems here. You cannot call processAccessoryEvent() with a const pointer as it needs a non-const pointer. Simply remove "const" in this parameter list.

Reply:
OK